docs/director/03-tenancy.md

```diff
@@ -0,0 +1,41 @@
+# Tenancy
+Tenant is the object that owns resources. 
+The Compass Director is a multi-tenant service.
```

Comment:
```suggestion
Compass Director is a multi-tenant service, which means that a single instance of Director is shared among different tenants (clients), but their data is isolated.
```

Reply:
I wonder if we should move this sentence to the next section 🤔 What is your opinion?